wordpress-mobile/WordPress-iOS
# History / Revisions: CoreData model version 85 and Blog authors

Refs. #10304 

This PR adds a new *CoreData model version*: **85**

The Revision Cell needs to display the Author avatar and the username.

Following the [Android approach](https://github.com/wordpress-mobile/WordPress-Android/pull/8465) I saw the app already uses that `/sites/$site/users` endpoint when a *Blog* is synced.
Speaking with @aerych the best solution was to add a new _Entity_ to store the authors within that response block. This will allow us to sync only the _Revisions_ and have all the authors informations we need before display the Revisions list.

The new entity is called `BlogAuthor`.
The `Blog` entity has a new property _authors_.

**To test:**
- Install this PR over a pre installed version which contains the previous data model version to test the migration
- Select different blogs to see if everything works ok
